### PR TITLE
Reporting rates report

### PR DESCRIPTION
**Pull request by lwyszomi**

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* custom/ewsghana/reports/__init__.py, line 49 - E124 closing bracket does not match visual indentation
* custom/ewsghana/reports/specific_reports/reporting_rates.py, line 16 - E302 expected 2 blank lines, found 1
* custom/ewsghana/reports/specific_reports/reporting_rates.py, line 39 - E226 missing whitespace around arithmetic operator
* custom/ewsghana/reports/specific_reports/reporting_rates.py, line 309 - W292 no newline at end of file
* custom/ewsghana/utils.py, line 59 - W292 no newline at end of file

File: custom/ewsghana/reports/specific_reports/reporting_rates.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+from custom.ewsghana import StockLevelsReport
+from custom.ewsghana.reports import MultiReport, EWSData
+from casexml.apps.stock.models import StockTransaction
+from custom.ewsghana.utils import calculate_last_period, get_supply_points
+from corehq.apps.reports.filters.dates import DatespanFilter
+from custom.ilsgateway.tanzania import make_url
+from custom.ilsgateway.tanzania.reports.utils import link_format
+from django.utils.translation import ugettext as _
+
+class AlertsData(EWSData):
```
Comment (dimagimon) [inline custom/ewsghana/reports/specific_reports/reporting_rates.py]: E302 expected 2 blank lines, found 1

File: custom/ewsghana/reports/specific_reports/reporting_rates.py [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+from custom.ewsghana.reports import MultiReport, EWSData
+from casexml.apps.stock.models import StockTransaction
+from custom.ewsghana.utils import calculate_last_period, get_supply_points
+from corehq.apps.reports.filters.dates import DatespanFilter
+from custom.ilsgateway.tanzania import make_url
+from custom.ilsgateway.tanzania.reports.utils import link_format
+from django.utils.translation import ugettext as _
+
+
+#TODO Implement this when alerts (moving from EWS) will be finished
```
Comment (dimagimon) [inline custom/ewsghana/reports/specific_reports/reporting_rates.py]: E265 block comment should start with '# '

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by kkrampa** [comment]
@modonnell729

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by lwyszomi** [comment]
@czue @twymer

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by twymer** [comment]
Mike has poked around with this some on staging and it seemed okay. Could definitely use some performance tuning, though, but not slow enough to be blocking.

**Pull request merged**